newspack-nelson/inc/child-typography.php

```diff
@@ -37,20 +37,21 @@ function newspack_nelson_custom_typography_css() {
 			.nav3,
 			.mobile-menu-toggle,
 			.accent-header,
+			.widgettitle,
```

Comment:
This should target the sidebar specifically, like: 

```suggestion
			#secondary .widgettitle,
```

... since the uppercase styles are not applied to widget areas like the slide-out sidebar, or when widgets appear in the mobile menu.